dataplane/standalone/saiserver/acl.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package saiserver
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openconfig/lemming/dataplane/forwarding/fwdconfig"
+	"github.com/openconfig/lemming/dataplane/standalone/saiserver/attrmgr"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	saipb "github.com/openconfig/lemming/dataplane/standalone/proto"
+	fwdpb "github.com/openconfig/lemming/proto/forwarding"
+)
+
+type aclDataplaneAPI interface {
+	ID() string
+	TableCreate(context.Context, *fwdpb.TableCreateRequest) (*fwdpb.TableCreateReply, error)
+	TableEntryAdd(context.Context, *fwdpb.TableEntryAddRequest) (*fwdpb.TableEntryAddReply, error)
+}
+
+type groupBank struct {
+	groupID  string
+	bank     int
+	memberID uint64
```

Comment:
What's the purpose of memberID?

Reply:
need to save the  id of the member in case it gets deleted later